### PR TITLE
vet-clinic: day-2: queries-aggregate-groupBy

### PR DESCRIPTION
**Pull request by Shubh-Dev**
UPDATE!
Wrong entries were corrected as suggested by the respected Reviewer.

Hello Dear Reviewer,
Completed day 2 activities.
Let me know your views.

Images:

columnadd
FTMiIieagAEGZaA
mon
plantmon
Screenshot from 2022-08-09 14-15-42
Screenshot from 2022-08-09 14-18-47
Screenshot from 2022-08-09 14-24-25
Screenshot from 2022-08-09 14-26-38
Screenshot from 2022-08-09 14-50-16
Screenshot from 2022-08-09 15-01-28
Screenshot from 2022-08-09 15-12-46
Screenshot from 2022-08-09 15-22-46
Screenshot from 2022-08-09 15-26-40
Screenshot from 2022-08-09 15-33-35
Screenshot from 2022-08-09 15-39-25
Screenshot from 2022-08-09 15-51-14
Screenshot from 2022-08-09 16-29-05

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -12,3 +12,27 @@ VALUES ('Pikachu', '2021-01-7', 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
 VALUES ('Devimon', '2017-05-12', 5, true, 11);
 
+-- day-2 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '2020-02-08', 0, false, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', '2021-11-15', 2, true, 5.7);
```
Comment (omar-muhamad) [inline data.sql]: - [ ] The weight of these two animals is supposed to be entered as a negative number as mentioned in the project requirements, So please fix that.

image

File: data.sql [hunk 2]
```diff
@@ -12,3 +12,27 @@ VALUES ('Pikachu', '2021-01-7', 1, false, 15.04);
...
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Squirtle', '1993-04-02', 3, false, 12.13);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Angemon', '2005-06-12', 1, true, 45);
```
Comment (omar-muhamad) [inline data.sql]: - [ ] Same here, The weight of these two animals is supposed to be entered as a negative number as mentioned in the project requirements, So please fix that.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Shubh-Dev,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Shubh-Dev 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**